processing/src/main/java/io/druid/segment/column/DoublesColumn.java

```diff
@@ -28,10 +29,12 @@
 public class DoublesColumn implements GenericColumn
```

Comment:
I suggest to have two separate classes, `DoublesColumnWithNulls` and `DoublesColumnWithoutNulls` (that's a rename of this class. Or just keep it's method. Or choose other.) And in non-null version `isNull()` implementation is `return false;`. Need the best optimization for the common case.

Reply:
added a check for emptyImmutableBitmap, isNull now also checks if the bitmap is empty or not. 
Also, I think we should optimize bitmap.get instead when the bitmap is empty to always return false, in case that is already not optimized, instead of making different classes at many places just for this optimization.

Reply:
It won't allow JVM to resolve this method call result statically and throw away branches in the upstream calling code. So it's very important to make the body of method *just* `return false;`.

One class could be extend another, overriding just isNull() method, so not so much code is copied and added.

Reply:
done.